src/frontend.md

```diff
@@ -221,6 +221,24 @@ export function fakeTicket() {}
 
 1. If some color is repeated several times in code, extract it to theme.
 
+## Disposition of components
+
+- Prefer using local components `(host|mf)/src/components` with a few exceptions
+- Use UI Kit distribution if a component represents a design token (1) [websome](<https://www.figma.com/file/I8WglqziPB5pYlFIqF43JE/Client-UI-Kit-(Web)?node-id=0-1&t=MuFhhOjUXP4sFRr2-0>) or matches with [MUI](https://mui.com/material-ui/)
```

Comment:
UI-kit. As I remember, we have a de facto rule of triple use. I'd like to add something to the Ui-kit and I'll receive a question how many repos will use it?

Reply:
It was before. I deleted it :) 
Let's assume you add a generic component used only in your repo. 
We know exactly what to place in UI Kit if we have the designer's UI kit and MUI as a roadmap,